listenbrainz_spark/recommendations/tests/test_dataframe.py

```diff
@@ -0,0 +1,152 @@
+import uuid
+import unittest
+from datetime import datetime
+
+import listenbrainz_spark
+from listenbrainz_spark.tests import SparkTestCase
+import listenbrainz_spark.recommendations.create_dataframes as dataframe
+from listenbrainz_spark import schema, utils, config, path, hdfs_connection
+
+from pyspark.sql import Row
+
+# path used in between test functions of this class
+LISTENS_PATH = '/test/listens/' # HDFS path to listens
+MAPPING_PATH = '/test/mapping.parquet'
+MAPPED_LISTENS_PATH = '/test/mapped_listens.parquet'
+
+DATE = datetime.utcnow()
```

Comment:
this variable being outside the test class and in constant style seems like an antipattern to me. I think we should just set it inside the test case as a member variable.